kolibri/plugins/user_profile/assets/src/views/ChangeFacility/index.vue

```diff
@@ -1,62 +1,30 @@
 <template>
 
-  <CoreBase
```

Comment:
:tada:

What are your impressions of the new pages architecture in the context of using it together with the state machine?